transifex/native/parsing.py

```diff
@@ -151,11 +165,18 @@ def extract_strings(self, src, origin=None):
             tree = ast.parse(src)
             visitor = CallDetectionVisitor(self._functions)
             visitor.visit(tree)
-            return parse_source_strings(visitor.function_calls)
+            source_strings, linenos = parse_source_strings(
+                visitor.function_calls)
```

Comment:
Relatively minor:

This has to do with the implementation of `parse_source_strings`.

Instead of:

```python
def parse_source_strings(...):
   return ['a', 'b'], ['A', 'B']

for lowercase, uppercase in zip(*parse_source_strings(...)):
    ...
```

Why not:

```python
def parse_source_strings(...):
    return [('a', 'A'), ('b', 'B')]

for lowercase, uppercase in parse_source_strings(...):
    ...
```

ie why should `parse_source_strings` return 2 identically sized arrays and not one array of 2-tuples?